Bigtable/src/BigtableClient.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
What are your thoughts on introducing this in a follow up PR? It has been nice to review this in small chunks.